guideline.md

```diff
@@ -0,0 +1,126 @@
+No approval required
+
+For some things we want to ensure chapters can get the quickest access possible, this means the requirement for pre-approval is removed. The chapter must still create a ticket on our platform with relevant information but can immediately send an invoice to the PySV (with the PyLadies email copied in) These items include:
+- Up to 200 EUR for stickers per chapter per year
+- A roll-up banner (one-time purchase) for a chapter
+- Shared subscriptions - i.e. Restream (these subscriptions should have been agreed upon by the chapters beforehand)
+
+These quick approvals apply to all existing and future chapters who have had at least one event or are attending PyCon DE as a group. Stickers for chapters might be ordered together (to reduce costs) and this should be organised between the chapters.
+
+### Single requests requiring approval
+
+For most requests approval will be required. At least one member of the approval group who is not part of the requesting chapter should approve a request. If a member of the approval group is not sure about a request they may request that a second approval is required (but the maximum number of approvals required should be 2).
+
+Members of the approval group may ask questions regarding the funding (or point groups to existing resources that are shared by the chapters). Approvals should happen in a timely manner (see approval group description below). If a PyLady requesting funds feels the process (considering everyone involved is a volunteer) is taking too long they should raise this with the PyLadies Germany Spokesperson.
+
+We expect chapters with multiple groups within it to have transparency with one another and to help us achieve an equitable way of distributing funds.
+
+### Request for larger amounts
+
+Some chapters might prefer to request approval for a larger amount in one go and manage the details of how it is spent internally (specifics still need to be reported and individual invoices sent). This could be for example be requesting a proportion of the total grant for the year upfront or for a single initiative which will include multiple costs. Ideally, such initiatives may be special programs that are indicated as planned by the chapter or chapters and presented, whenever possible, at the beginning of the year in order to plan the annual budget. After approval, the chapter will be able to get reimbursements for up to this amount without the need for further approval.
+
+These chapters should already be established, with a track record of events they have run and (some) consistency in the organizing team (i.e. it should not be a whole new group who have recently taken over a chapter). It would be expected that this group would not receive approval for applications for further funds unless we have leftover budget (towards year end).
```

Comment:
Should we say that for this kind of request a group needs to have run at least 5 events in the past 12 months? It's an arbitrary number but I'm trying to think from the perspective of a new chapter to give them more clarity on when they would be able to make these kinds of requests.

Reply:
That's a great point, we want transparency and explicitness on what we mean!